lib/styles/base/_base.scss

```diff
@@ -106,43 +106,52 @@ body {
   color: $neeto-ui-gray-800;
 }
 
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
+  letter-spacing: -0.5px;
+}
+
 h1 {
   font-size: $neeto-ui-text-3xl;
   font-weight: $neeto-ui-font-semibold;
-  line-height: $neeto-ui-leading-tight;
+  line-height: 42px;
```

Comment:
Regarding `line-height: 42px;`, let’s use percentage values (here 1.31 or we can use the default `$neeto-ui-leading-snug: 1.375;`) for line heights so that we can avoid redefining `line-height` based on the viewport.

Reply:
For that to work, we need to redefine the font sizes based on the viewport right?